kasa/tests/fixtures/smart/child/T315(EU)_1.0_1.7.0.json

```diff
@@ -0,0 +1,537 @@
+{
+   "component_nego" : {
```

Comment:
This is added manually for now to make the tests pass. dump_devinfo should be changed to request the components & perform queries based on the implemented modules.